includes/learnopengl/model.h

```diff
@@ -192,6 +266,13 @@ class Model
             {   // if texture hasn't been loaded already, load it
                 Texture texture;
                 texture.id = TextureFromFile(str.C_Str(), this->directory);
+                if (texture.id == noDiffuse)
+                {
+                    if (typeName == "texture_specular")
```

Comment:
You can also put this after the normal as an "else if"
This come from my long experience with Gamemaker 8.0 lol
I'm not sure if compiler optimize it, but if one pass the other don't need to be checked.

Reply:
Fixed 👍